install/ui/src/freeipa/rpc.js

```diff
@@ -219,6 +225,7 @@ rpc.command = function(spec) {
 
         for (var i=0,l=msgs.length; i<l; i++) {
             var msg = lang.clone(msgs[i]);
+            if (that.suppress_warnings.indexOf(msg.code) > -1) break;
```

Comment:
Shouldn't it be continue instead of break? So that it will process also other possible msgs which you don't want to suppress.